idpartner-hosted/docs/prerequisites.md

```diff
@@ -0,0 +1,19 @@
+# Prerequisites
+Before proceeding, ensure the following tools are installed on your system as they are essential for completing the setup.
+
+| Tool    | Instructions                                                           | Verification Command | Supported Version  |
+| ------- | ---------------------------------------------------------------------- | -------------------- | ------------------ |
+| ngrok   | [Download from ngrok](https://ngrok.com/download)                      | `ngrok version`      | 3.1.0 and above    |
+| nodejs  | [Download from Node.js](https://nodejs.org/en/download/)               | `node -v`            | v16.16.0 and above |
+| yarn    | [Download from Yarn](https://classic.yarnpkg.com/lang/en/docs/install) | `yarn --version`     | 1.18.0 and above   |
+
+Install dependencies. Run `yarn install` and make sure it completes without errors.
+
+## Install Dependencies
+After installing the tools, run the following command to install the necessary dependencies. Ensure that this process completes without any errors.
+```bash
+yarn install
```

Comment:
Rod made me change to npm, in template download, won't it be the same?

Reply:
Updated to use npm